src/Lykke.Job.BlockchainCashoutProcessor/Mappers/MappingExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Lykke.Job.BlockchainCashoutProcessor.Core.Domain;
+using Lykke.Job.BlockchainCashoutProcessor.Contract;
+using Lykke.Job.BlockchainOperationsExecutor.Contract;
+
+namespace Lykke.Job.BlockchainCashoutProcessor.Mappers
+{
+    public static class MappingExtensions
+    {
+        public static CashoutResult MapToChashoutProcessResult(this OperationExecutionErrorCode source)
+        {
+            switch (source)
+            {
+                case OperationExecutionErrorCode.Unknown:
+                    return CashoutResult.Unknown;
+                case OperationExecutionErrorCode.AmountTooSmall:
+                    return CashoutResult.AmountTooSmall;
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(source), source, null);
+            }
+        }
+
+
+        public static ChashoutErrorCode MapToChashoutProcessErrorCode(this CashoutResult source)
```

Comment:
Its strange to map CashoutResult to CashoutErrorCode.

Reply:
First time OperationExecutionErrorCode mapped to CashoutResult error code then CashoutResult mapped to CashoutErrorCode.

CashoutResult i used as cashout processor error code.

Reply:
So you have to map error code to error code. Not error code to result

Reply:
For error code to result i used MapToCashoutProcessResult extension